testcases/MachineConfig.py

```diff
@@ -141,6 +142,10 @@ def callConfig(self, key):
                 self.fail(status)
             if not setup:
                 self.skipTest("Not implemented for other OS settings")
+    
+    def execute_io_enlarge_capacity(self):
+        cureent_io_enlarge = EBMCHostManagement.get_current_ioadapter_enlarged_capacity()
+        log.info("current io enlarger capacity: {cureent_io_enlarge}")
 
```

Comment:
get a new parameter in Class CecConfig()   i.e self.iocapcity  for ioenlargecapacity and if given   call set()  and validate values set... using get()